### ISSUE TITLE
Inconsistent highlighting in haskell

### ISSUE THREAD

**Issue by antonand14** [sublimehq/Packages#2101]
As, the title says. Problem can be seen in attached screenshot at rows 136-137, 147-148, 163-164. Linux version 3.2.1 build 3207
![Screenshot from 2019-09-23 21-58-51@2x](https://user-images.githubusercontent.com/54798762/65461336-f1afe880-de53-11e9-9fa2-7b21693930ce.png)

```Haskell
shuffle' :: [Double] -> Deck -> (Card,Deck) 
shuffle' (random:doubles) deck 
    | size deck == 1 = (deck !! 0,[])
    | otherwise      = ((deck !! index), removeElement deck index)
        where index  = ceiling (random * size deck) - 1
```

```Haskell
playBank' :: Deck -> Hand -> Hand
playBank' deck bankHand
    | value bankHand < 16 = playBank' deck' bankHand'
    | otherwise           = bankHand
        where (deck', bankHand') = draw deck bankHand
```

**Comment by jfcherng** [comment]
it will be good to provide source code along with the screenshot.

**Comment by antonand14** [comment]
> it will be good to provide source codes along with the screenshot.

i've added two examples, thanks for the heads up

**Comment by deathaxe** [comment]
Duplicate of #1320

Fixed by https://github.com/sublimehq/Packages/commit/98638f763ad4845a4063af80fa9b1cd9a93f39c3 which is already merged to master.

**Comment by antonand14** [comment]
> Duplicate of #497, #1320
> 
> Fixed by [98638f7](https://github.com/sublimehq/Packages/commit/98638f763ad4845a4063af80fa9b1cd9a93f39c3) which is already merged to master

Sorry i'm new to github so i dont know if i am missing something but i am still seeing this issue.

**Comment by deathaxe** [comment]
No problem. The fix is not yet part of ST 3207 (or ST 3210). Will hopefully ship with one of the next ST dev builds.

**Comment by antonand14** [comment]
> Duplicate of #497, #1320
> 
> Fixed by [98638f7](https://github.com/sublimehq/Packages/commit/98638f763ad4845a4063af80fa9b1cd9a93f39c3) which is already merged to maste

**Comment by antonand14** [comment]
> No problem. The fix is not yet part of ST 3207 (or ST 3210). Will hopefully ship with one of the next ST dev builds.

Oh i see, alright thats great. thanks for the info
